lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filters/edit-service-task-filter-cloud.component.ts

```diff
@@ -0,0 +1,279 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnChanges, Output, EventEmitter, SimpleChanges, OnInit, OnDestroy } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { DateAdapter } from '@angular/material/core';
+import { MatDialog } from '@angular/material/dialog';
+import { filter, takeUntil, finalize, switchMap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { Moment } from 'moment';
+
+import { TaskFilterProperties, TaskFilterAction, ServiceTaskFilterCloudModel } from '../../models/filter-cloud.model';
+import { TaskFilterDialogCloudComponent } from '../task-filter-dialog/task-filter-dialog-cloud.component';
+import { TranslationService, UserPreferencesService } from '@alfresco/adf-core';
+import { AppsProcessCloudService } from '../../../../app/services/apps-process-cloud.service';
+import { TaskCloudService } from '../../../services/task-cloud.service';
+import { ServiceTaskFilterCloudService } from '../../services/service-task-filter-cloud.service';
+import { BaseEditTaskFilterCloudComponent } from './base-edit-task-filter-cloud.component';
+
+@Component({
+    selector: 'adf-cloud-edit-service-task-filter',
+    templateUrl: './base-edit-task-filter-cloud.component.html',
+    styleUrls: ['./base-edit-task-filter-cloud.component.scss']
+})
+export class EditServiceTaskFilterCloudComponent extends BaseEditTaskFilterCloudComponent implements OnInit, OnChanges, OnDestroy {
+
+    public static DEFAULT_TASK_FILTER_PROPERTIES = ['appName', 'activityName', 'status', 'sort', 'order'];
+    public static DEFAULT_TASK_SORT_PROPERTIES = ['id', 'name', 'startedDate', 'completedDate'];
+    public static DEFAULT_TASK_STATUS_PROPERTIES = [
+        { label: 'ALL', value: '' },
+        { label: 'STARTED', value: 'STARTED' },
+        { label: 'COMPLETED', value: 'COMPLETED' },
+        { label: 'CANCELLED', value: 'CANCELLED' },
+        { label: 'ERROR', value: 'ERROR' }
+    ];
+
+    /** Emitted when a task filter property changes. */
+    @Output()
+    filterChange: EventEmitter<ServiceTaskFilterCloudModel> = new EventEmitter();
+
+    taskFilter: ServiceTaskFilterCloudModel;
+    changedTaskFilter: ServiceTaskFilterCloudModel;
+
+    constructor(
+        protected formBuilder: FormBuilder,
+        public dialog: MatDialog,
+        private translateService: TranslationService,
+        private serviceTaskFilterCloudService: ServiceTaskFilterCloudService,
+        protected dateAdapter: DateAdapter<Moment>,
+        protected userPreferencesService: UserPreferencesService,
+        protected appsProcessCloudService: AppsProcessCloudService,
+        protected taskCloudService: TaskCloudService) {
+        super(formBuilder, dateAdapter, userPreferencesService, appsProcessCloudService, taskCloudService);
+    }
+
+    ngOnInit() {
+       super.ngOnInit();
```

Comment:
you don't need to declare a method that calls base method, remove entire ngOnInit instead